oneplus/views.py

```diff
@@ -33,7 +33,7 @@
 from communication.utils import report_user_post
 from organisation.models import School, Course
 from core.models import Class
-from oneplusmvp.settings import OPEN_SCHOOL_NAME, OPEN_CLASS_NAME, OPEN_COURSE_NAME
+from oneplusmvp.settings import GRADE_10_COURSE_NAME, GRADE_11_COURSE_NAME, GRADE_10_OPEN_CLASS_NAME, GRADE_11_OPEN_CLASS_NAME
```

Comment:
It is better to have settings.GRADE_10_... further down. The way it is now, if one of them is not defined the site won't load because there is a general load failure. Please change it.